src/actions/users.js

```diff
@@ -224,6 +224,8 @@ export function getProfiles(offset, limit = General.PROFILE_CHUNK_SIZE) {
             return null;
         }
 
+        const profiles = profileListToMap(profilesList);
```

Comment:
I think it's better to be stored as a map in the store for ease of access but I could probably be convinced to store it as an array if I was presented with some good arguments for it. Either way it's a bit out of scope for this ticket and we can revisit when I start working on the store itself